projects/onto-search/src/test.ts

```diff
@@ -0,0 +1,26 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
```

Comment:
Is this generated in some way? Do we need it? We discussed that we are going to write cypress tests and not unit. Right?